src/pages/Home/HardwareCard.tsx

```diff
@@ -70,7 +70,7 @@ export const HardwareCard: FC = () => {
         <div className="flex w-1/2 flex-col">
           <h6 className="text-sm text-gray-200">{t("hardware.disk_usage")}</h6>
           <p className="flex">
-            {hddUsedGB} / {hddTotalGB} GB ({hddPercentUsed} %)
+            {hddUsedGB} /{hddTotalGB} GB ({hddPercentUsed} %)
```

Comment:
should we really remove the white space here?

would look weird right?

Before: `20 / 20 GB (100%)`
After: `20 /20 GB (100%)`

Reply:
Thanks, good catch! Fixed